code/datums/station_traits/negative_traits.dm

```diff
@@ -343,7 +343,7 @@
 
 /datum/station_trait/random_event_weight_modifier/dust_storms
 	name = "Dust Stormfront"
-	report_message = "The space around your station is clouded by heavy pockets of space dust. Expect an increased likelyhood of space dust storms damaging the station hull."
+	report_message = "The space around your station is clouded by heavy pockets of space dust. Expect an increased likelihood of space dust storms damaging the station hull."
```

Comment:
again remember, this is a core file, so these edits need to be marked (unless this is restoring to tg)

Reply:
It's resetting the file back to TG